NetUtils.py

```diff
@@ -334,8 +341,10 @@ def as_network_message(self) -> dict:
         add_json_text(parts, ". ")
         if self.found:
             add_json_text(parts, "(found)", type="color", color="green")
+        elif self.prioritized:
+            add_json_text(parts, "(priority)", type="color", color="red")
         else:
-            add_json_text(parts, "(not found)", type="color", color="red")
+            add_json_text(parts, "(non-priority)", type="color", color="red")
```

Comment:
should priority and non-priority have the same color? Feels like a different color would be nice

Reply:
Thought about it, but not sure what color (esp regarding colorblind friendly color choices and stuff)

Reply:
I know nothing about color blindness, but it feels like a non priority hint would fit well in the middle of the "standard" red->yellow->green pattern

Reply:
oof. Thankfully the red isn't my addition, so not in my scope to worry about.